src/client/components/componentWithSliders/modalSavePressetesWindow/modalSavePressetesWindow/modalSavePressetesWindow.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modalSavePressetesWindow.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import SavePressetsToBDAndCancelButton from './savePressetsToBDAndCancelButton';
+
+const cancelWindowIcon = <FontAwesomeIcon icon={faTimes} />;
```

Comment:
You have file for icons, you can move it there

Reply:
ok